liblumen_codegen/src/llvm/builder.rs

```diff
@@ -280,22 +280,22 @@ impl<'ctx> ModuleBuilder<'ctx> {
         unsafe { LLVMBuildRet(self.builder, ret) }
     }
 
-    pub fn add_constant(
+    pub fn build_constant(
```

Comment:
Why the shift from `add` to `build`?   Are all the builders guaranteed to not add to an outer scope right away?

Reply:
As a convention I've been trying to stick to using the `build_` suffix on any builder APIs, but these I misnamed early on and hadn't gotten around to addressing it.